h/static/scripts/group-forms/components/forms/FormContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ComponentChildren } from 'preact';
+
+export type FormContainerProps = {
+  title: string;
+  children: ComponentChildren;
+};
+
+/** A container for a form with a title. */
+export default function FormContainer({ children, title }: FormContainerProps) {
```

Comment:
The document title currently doesn't update when switching between the group settings and members pages. This can be added in future with an effect in this component.